types/ramda/index.d.ts

```diff
@@ -225,9 +225,9 @@ declare namespace R {
         /**
          * Returns a new list containing the contents of the given list, followed by the given element.
          */
-        append<T, U>(el: U, list: T[]): Array<(T & U)>;
-        append<U>(el: U): <T>(list: T[]) => Array<(T & U)>;
         append<U>(el: U): <T>(list: T[]) => Array<(T & U)>;
+        append<T, U>(el: U, list: T[]): Array<(T & U)>;
```

Comment:
Why is this not `Array<T | U>`? But I guess it was already written this way?

wat

Reply:
convenience!